openerp/openupgrade/doc/source/modules70-80.rst

```diff
@@ -416,7 +416,7 @@ Status :
 +-----------------------------------+-----------------------------------+
 |share                              | Nothing to do                     |
 +-----------------------------------+-----------------------------------+
-|stock                              | Done (experimental)               |
+|stock                              | Done                              |
```

Comment:
Uhm, not for multi-company, but with multi-warehouse I didn't have any special problem